Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -103,10 +103,10 @@ public final class System {
                          (name: "Gwei",  code: "gwei", decimals:  9, symbol: BlockChainDB.Model.lookupSymbol ("gwei")),
                          (name: "Ether", code: "eth",  decimals: 18, symbol: BlockChainDB.Model.lookupSymbol ("eth"))]),
 
-        (id: System.makeCurrencyIdentifierERC20 ("ethereum-mainnet", BlockChainDB.Model.addressBRDMainnet), name: "BRD Token", code: "BRD", type: "erc20", blockchainID: "ethereum-mainnet",
+        (id: System.makeCurrencyIdentifierERC20 ("ethereum-mainnet", BlockChainDB.Model.addressBRDMainnet), name: "BRD Token", code: "brd", type: "erc20", blockchainID: "ethereum-mainnet",
          address: BlockChainDB.Model.addressBRDMainnet, verified: true,
-         demoninations: [(name: "BRD Token INT", code: "BRDI",  decimals:  0, symbol: "brdi"),
-                         (name: "BRD Token",     code: "BRD",   decimals: 18, symbol: "brd")]),
+         demoninations: [(name: "BRD Token INT", code: "brdi",  decimals:  0, symbol: "brdi"),
```

Comment:
Any issue with this (code formatting) diverging from `currencyToDefaultBaseUnit` and `asBlockChainDBModelCurrency` behaviour?

Reply:
Looks like it.  This:
```
                let code         = code.uppercased()
```
should be 'lowercased()'?

Reply:
> Looks like it. This:
> 
> ```
>                 let code         = code.uppercased()
> ```
> 
> should be 'lowercased()'?

Can you update the Swift? I'll mirror in Java. You have the "mental model" here as to how these should be.

Reply:
And these:
```
                            (name: "\(code)_INTEGER",
                                code: "\(code)_INTEGER",
                                decimals: 0,
                                symbol: "\(code)I"),   // BRDI -> BaseUnit

```
should have the code and symbol be "\(code)i" ?